client/me/purchases/cancel-purchase/index.jsx

```diff
@@ -63,12 +64,52 @@ const CancelPurchase = React.createClass( {
 		page.redirect( redirectPath );
 	},
 
+	renderFooterText() {
+		const purchase = getPurchase( this.props ),
+			{ refundText, renewDate } = purchase;
+
+		if ( isRefundable( purchase ) ) {
+			return this.translate( '%(refundText)s to be refunded', {
+				args: { refundText },
+				context: 'refundText is of the form "[currency-symbol][amount]" i.e. "$20"'
```

Comment:
We could probably use a different name for `refundText` to better convey what it actually contains:

```javascript
renderFooterText() {
		const purchase = getPurchase( this.props ),
			{ refundText: amount, renewDate } = purchase;

		if ( isRefundable( purchase ) ) {
			return this.translate( '%(amount)s to be refunded', {
				args: { amount },
				context: 'amount is of the form "[currency-symbol][amount]" i.e. "$20"'
			} );
		}
```